pom.xml

```diff
@@ -293,6 +325,14 @@
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>21</source>
+                    <target>21</target>
```

Comment:
Why do we need to set the versions here when we already define them for the compilter plugin in the properties?

Reply:
removed